src/System.Drawing.Common/tests/Text/PrivateFontCollectionTests.cs

```diff
@@ -149,19 +149,12 @@ public void AddFontFile_LongFilePath_ThrowsException()
             }
         }
 
-        [ActiveIssue(20884, TestPlatforms.AnyUnix)]
         [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void AddFontFile_Directory_ThrowsExternalException()
+        public void AddFontFile_Directory_ThrowsFileNotFoundException()
```

Comment:
On the netfx leg this is now failing with:
```
Assert.Throws() Failure
Expected: typeof(System.IO.FileNotFoundException)
Actual:   typeof(System.Runtime.InteropServices.ExternalException): A generic error occurred in GDI+.
---- System.Runtime.InteropServices.ExternalException : A generic error occurred in GDI+.
```

Reply:
Thanks, I updated the test

Reply:
Thanks!